src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class WinningLotto {
+    private static final int LOTTO_SIZE = 6;
+    private static final int LOTTO_MIN_NUMBER = 1;
+    private static final int LOTTO_MAX_NUMBER = 45;
+    private static final String DELIMITER = ",";
+    private final Set<Integer> winningNumbers;
```

Comment:
```suggestion
    private final Lotto winningNumbers;
```
Integer 컬렉션도 로또 객체로 볼 수 있지 않을까요?

Reply:
해당 부분을 수정하다보니 WinngingLotto라는 객체가 필요가 없어보여 해당부분을 합쳤습니다!